src/templates/contributors/article-list.md

```diff
@@ -0,0 +1,26 @@
+project_path: /web/_project.yaml
+book_path: /web/resources/_book.yaml
+
+{# wf_auto_generated #}
+{# wf_updated_on: 1900-01-01 #}
+{# wf_published_on: 1900-01-01 #}
+
+# {{{ title }}} {: .page-title }
+
+{{#each articles}}
+
+## {{ title }}
+<div class="attempt-right">
+  {{#if image}}
```

Comment:
If there's no image, let's just skip it altogether, it adds a bunch of extra page weight and won't look great on mobile.

Reply:
Done